internal/step/plugin/provider.go

```diff
@@ -971,7 +971,7 @@ func (r *runningStep) ForceClose() error {
 	}
 	// Wait for the run to finish to ensure that it's not running after closing.
 	r.wg.Wait()
-	r.logger.Warningf("Step %s/%s force closed.", r.runID, r.pluginStepID)
+	r.logger.Infof("Step %s/%s force closed.", r.runID, r.pluginStepID)
```

Comment:
As-designed, I believe that it may still call the force close as a final cleanup step. So, without design changes, this is the best minimum viable change to reduce unnecessary warnings. If desired, I can create an issue for this.

Reply:
"Without design changes" the core question here is whether it's really an "unnecessary warning"? If you can't be sure from context, then, yeah, a followup might be worthwhile although I doubt it'll bubble up to the top of the backlog any time soon. 😆